### PR TITLE
Verify macOS app bundle contains dependencies

### PR DESCRIPTION
**Pull request by phw**
# Summary
On macOS builds verify:

- The `picard-run` binary can be executed
- `picard-run --long-version` reports support for both astrcmp C and libdiscid
- `fpcalc` binary is included and executable

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [x] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-XXX](https://tickets.metabrainz.org/browse/PICARD-XXX)

# Solution

# Action

### CODE REVIEW

**Comment by phw** [comment]
Builds are working, see output in https://api.travis-ci.org/v3/job/592035633/log.txt

```
Picard 2.2.2.dev1 (osx_xcode9_20191001141545), Python 3.7.4, PyQt 5.13.1, Qt 5.13.1, Mutagen 1.42.0, Discid discid 1.2.0, libdiscid 0.6.2, astrcmp C, SSL Secure Transport, macOS Sierra (10.12)
fpcalc version 1.4.3
```

File: scripts/package-osx.sh [hunk 1]
```diff
@@ -37,14 +37,28 @@ if [ -f scripts/appledev.p12 ] && [ -n "$appledev_p12_password" ]; then
 fi
 
 cd dist
+
+# Create app bundle
 ditto -rsrc --arch x86_64 'MusicBrainz Picard.app' 'MusicBrainz Picard.tmp'
 rm -r 'MusicBrainz Picard.app'
 mv 'MusicBrainz Picard.tmp' 'MusicBrainz Picard.app'
 [ "$CODESIGN" = '1' ] && codesign --keychain $KEYCHAIN_PATH --verify --verbose --deep --sign "$CERTIFICATE_NAME" 'MusicBrainz Picard.app'
+
```
Comment (samj1912) [inline scripts/package-osx.sh]:
```suggestion
Just testing out a suggest
```

Comment (samj1912) [reply scripts/package-osx.sh]: Interesting. @zas @phw did you know about this?

https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request

See point 6

**Comment by phw** [comment]
@samj1912 No, but that's pretty cool. I have to try it next time

**Pull request merged**